velox/core/QueryConfig.h

```diff
@@ -354,6 +354,15 @@ class QueryConfig {
   static constexpr const char* kDriverCpuTimeSliceLimitMs =
       "driver_cpu_time_slice_limit_ms";
 
+  /// The max bytes number of normalized keys for each row in the prefixsort.
```

Comment:
@skadilover : There could be cases of sorting with multiple keys, e.g. In WindowNode, we sort with a combination of a (partition + orderby) keys. In that case, should we use 2*uint32 instead ? In general, should number of keys * sizeof(uint32_t) be a good prefix size to use ?

Reply:
I think I haven’t fully understood the issues you are concerned about. Can you describe it in more detail?

> In general, should number of keys * sizeof(uint32_t) be a good prefix size to use

The prefix size depend on the types of keys，for example , sort keys are : c1_int32, c2 int64, the prefix size is 4 + 8 = 12.

This config item is the 'max limit' , as duckDB do not limit it ,  I think we can set it as max-int by default.